usr.sbin/pmcstat/pmcstat.c

```diff
@@ -1516,5 +1520,5 @@ main(int argc, char **argv)
 			    );
 	}
 
-	exit(EX_OK);
+	exit(exitcode);
```

Comment:
pmcstat(1) already uses <sysexits.h> values in various other places, so using EXIT_* is not consistent.  I can see arguments for both behaviors here as you might be profiling a failing command on purpose and the profiling didn't fail, just the executed command (which is what the current behavior tailors towards).  The other error returns in pmcstat (of which they are many) all deal with the profiling failing, not the status of the child process.  Arguably in our use case of SPEC we should really be running the tests without pmcstat initially to validate that they succeed (including validating output if necessary, not just if it cores) before trying to run them under pmcstat.

Reply:
I still think propagating the error would be nice - how about EX_OSERR? I'm not sure any of the sysexits really match, that one just seemed somewhat appropriate.